### PR TITLE
fix: GMP-393 - improve and fix some scripts and provide some output c…

### PR DESCRIPTION
**Pull request by mmichna**
…ustom actions parameters

### CODE REVIEW

**Review by rupertohpen — APPROVED** [review]

File: actions/terraform/tfm-plan/action.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ inputs:
   secret-key:
     description: "secret key"
     required: true
-  account:
+  account: #FIXME we can remove it because there is wrong order in script
```
Comment (mmichna) [inline actions/terraform/tfm-plan/action.yml:13]: This need be fixed asap because in scripts it is stored to: TFM_FOLDER :(

**Review by MartinMitro — APPROVED** [review]

**Comment by lucky-edu** [comment]
closing PR due to inactivity

**Comment by MartinMitro** [comment]
why closing it? it's Christmas period, so of course there is inactivity.

**Comment by mmichna** [comment]
@lucky-edu: @rupertohpen inform me that somebody from his team will check this PR - so i wait on your approvals. So PLS check it and then i will merge it - this PR contains very important improvements and we need discuss because there are also bugs in shared actions (maybe already fixed)

**Comment by mmichna** [comment]
@lucky-edu: maybe is better contact author of PR via Team before we will close some PR

**Pull request closed without merging**